bin/deploy-data/ood/form.yml

```diff
@@ -89,7 +45,4 @@ attributes:
     display: true
     help: |
       (Optional) Select the whisper model.
-    options:
-    - [ "default",  "large-v3" ]
-    - [ "v2",  "large-v2" ]
-    - [ "v3",  "large-v3" ]
\ No newline at end of file
+    options: #Will be filled with bin/deploy-data/update_supported_lang.py
```

Comment:
Should this be bin/deploy-data/update_ood_form_values.py?

Reply:
Thanks Teemu for noticing.